programs/ttp_oracle/src/processor.rs

```diff
@@ -43,20 +46,97 @@ impl Processor {
 
     Ok(())
   }
+
+  pub fn process_handle_response(accounts: &[AccountInfo], response: &Response) -> ProgramResult {
+    // clear the account data
```

Comment:
when making comments, lets use doc comments so it will show up in IDEs and auto generated docs
https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html